src/pe/export.rs

```diff
@@ -173,16 +200,16 @@ impl<'a> Reexport<'a> {
 #[derive(Debug, Default)]
 /// An exported symbol in this binary, contains synthetic data (name offset, etc., are computed)
 pub struct Export<'a> {
```

Comment:
So the export has every field optional? This doesn't seem right somehow, and i'm not sure what it means exactly for every field to be optional now.

I don't like this, and don't really think this is an appropriate API to expose to consumers; if its somehow:

```
Export {
  name: None,
  offset: None,
  rva: None
  reexport: None
}
```

Then the export is really just None, and the api/construction of exports should drop it, not push optionalness into the fields.